src/v2/conversions.rs

```diff
@@ -2276,17 +2266,9 @@ impl TryFrom<RejectReason> for super::types::RejectReason {
             reject_reason::Reason::NonExistentTokenId(token_id) => Self::NonExistentTokenId {
                 token_id: token_id.try_into()?,
             },
-            reject_reason::Reason::TokenHolderTransactionFailed(token_module_reject_reason) => {
-                Self::TokenModule(token_module_reject_reason.try_into()?)
-            }
-            reject_reason::Reason::TokenGovernanceTransactionFailed(token_module_reject_reason) => {
+            reject_reason::Reason::TokenUpdateTransactionFailed(token_module_reject_reason) => {
                 Self::TokenModule(token_module_reject_reason.try_into()?)
             }
```

Comment:
Otherwise the JSON serialization doesn't match.

```suggestion
            reject_reason::Reason::TokenUpdateTransactionFailed(token_module_reject_reason) => {
                Self::TokenUpdateTransactionFailed(token_module_reject_reason.try_into()?)
            }
```